### PR TITLE
test/new-e2e/tests/apm: skip vm test suite

### PR DESCRIPTION
**Pull request by ahmed-mez**
### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by pducolin — APPROVED** [review]

File: test/new-e2e/tests/apm/vm_test.go
Comment (pducolin) [inline test/new-e2e/tests/apm/vm_test.go:39]: **💬 suggestion**
You could create a JIRA card or use the incident-25117 as a reference, it helps rolling back the skip once addressed

Comment (ahmed-mez) [reply test/new-e2e/tests/apm/vm_test.go]: Good idea, thanks!

**Comment by KSerrania** [comment]
Force-merging without code owner approval to resolve #incident-25117. This is a PR that only skips a failing test, no expected side-effect.

**Pull request merged**
